### ISSUE TITLE
Experimental setup of the outfit-specific version 

### ISSUE THREAD

**Issue by tpzou** [jsnln/fite#1]
Hi @jsnln , congrats on this great work, and thanks a lot for open-sourcing this project!

I note that the experimental setup of the outfit-specific version was not detailed in the article. Could you describe it specifically? Such as training split and testing split of dataset?

**Comment by jsnln** [comment]
Thanks for your interest in our work. To run an outfit specific version, simply put a single outfit in the config file `xxx_subject_list.yaml` where `xxx` is the name of the experiment. Note that you should also specify the outfit in `step1.yaml` and `step2.yaml`.

The training and testing split are the official splits of ReSynth (https://pop.is.tue.mpg.de/index.html)

**Comment by tpzou** [comment]
Thanks for your quick reply! The per-outfit experiment in POP looks very similar to the outfit-specific version.  But I still can‘t find the detailed information in the [code](https://github.com/qianlim/POP) or paper. 

As there are 37 motion sequences of 03375_blazerlong in CAPE, which sequences do you use for the test set and which ones for the training set? Or is there another split way?

**Comment by jsnln** [comment]
In the download page of the CAPE (https://cape.is.tue.mpg.de/download.php) you can find a version of CAPE packed for POP, which defines a train/test split. We used this official split.
